src/app/api/movies/[movieId]/screenings/route.ts

```diff
@@ -0,0 +1,28 @@
+import { db } from '@/utils/db';
+import Screening from '@/models/screening.model';
+import screeningModel from '@/models/screening.model';
+import { NextResponse } from 'next/server';
+import { useParams } from 'next/navigation';
+db();
+
+export async function GET(request: any, { params }: any) {
+  const screenings = await Screening.find({ movie: params.movieId });
+  return new Response(JSON.stringify(screenings));
+}
+
+export async function POST(request: any, { params }: any) {
+  try {
+    const newScreening = new screeningModel({
+      salon: 'B',
+      time: '2023-05-25T12:00:00.685Z',
+      movie: params.movieId,
+      seats: 48,
+    });
```

Comment:
Is this code for creating dummy data, or what is the purpose of having hard-coded values in the API?